documents/motions/10.2022.3 - Disciplinary Committee.md

```diff
@@ -1,34 +1,40 @@
+      1. Being the confidential contact for Registered Speedcubers and WCA Staff, in case of important disciplinary and ethical matters that are related to WCA activities.
    3. Information and administration
       1. Informing the WCA Board on investigations, decisions, and disciplinary actions.
-      2. Publishing public information on the investigations, decisions, and disciplinary actions.
+      2. Publishing public information on the investigations, decisions, and disciplinary actions where necessary.
       3. Managing and administrating investigations, reports, decisions, and disciplinary actions.
       4. Keeping all information confidential that needs to stay confidential, based on internal and external guidelines
-2. All WCA Staff must fully cooperate with investigations by the WCA Disciplinary Committee.
-3. All Registered Speedcubers must fully cooperate with investigations by the WCA Disciplinary Committee.
-   1. All Registered Speedcubers should respond to emails sent on behalf of the WCA Disciplinary Committee within one week of each message.
-      1. A Registered Speedcuber must inform the WCA Disciplinary Committee or the WCA Board if they decide to stop responding to the WCA Disciplinary Committee and explain as to the reasons why.
-4. The WCA Disciplinary Committee will take the role of WCA Ethics Committee in case the WCA Ethics Committee is not constituted or for any reason non-functional.
-5. In case the WCA Disciplinary Committee is not constituted or for any reason non-functional, the WCA Ethics Committee will act as WCA Disciplinary Committee.
+   4. Document maintenance
+      1. Maintaining and publishing a Code of Ethics that all WCA Staff must abide by.
+      2. Maintaining and publishing a Code of Conduct that all WCA Members must abide by.
+3. All WCA Members must fully cooperate with investigations by the WCA Integrity Committee.
+   1. All WCA Members should respond to communications sent on behalf of the WCA Integrity Committee within one week of each message.
+      1. A WCA Member must inform the WCA Integrity Committee if they decide to stop responding to the WCA Integrity Committee and explain as to the reasons why.
+      2. The WCA Integrity Committee may place temporary sanctions on WCA Members who cease cooperating with the WCA Disciplinary Committee
+5. In case the WCA Integrity Committee is not constituted or for any reason non-functional, the WCA Appeals Committee will act as WCA Integrity Committee.
 6. Reducing conflicts of roles
-   1. There may be no overlap in members of the WCA Board, WCA Disciplinary Committee, and WCA Ethics Committee unless the Board is temporarily required to lead, due to that Committee not having a Leader.
+   1. There may be no overlap in members of the WCA Board, WCA Integrity Committee, and WCA Appeals Committee unless the Board is temporarily required to lead, due to that Committee not having a Leader.
```

Comment:
allow for the case that one of the committees is not constituted or functional - in that case the committee is not just led by the Board, the Board is the committee.